tdrs-backend/tdpservice/core/views.py

```diff
@@ -0,0 +1,18 @@
+"""Define core, generic views of the app."""
+import logging
+
+from rest_framework.response import Response
+from rest_framework.decorators import api_view
+
+logger = logging.getLogger()
+
+
+@api_view(['POST'])
+def write_logs(request):
+    """Pass request bodies to the system logger.
+
+    Mainly used to log client-side alerts and errors.
+    """
+    logger.info(request.data)
+
+    return Response('Success')
```

Comment:
Generic endpoint that accepts arbitrary json, logs it, and returns a generic `Success` response.

Reply:
Would it cause an issue with the frontend if we made this endpoint require authentication?

Reply:
Added